### ISSUE TITLE
can the compiler scale the output or keep it within a bounding box?

### ISSUE THREAD

**Issue by tedder** [PadLex/SvgToGcode#20]
I'm not seeing it, but given the first letter of SVG, can the output be scaled? I assume that would happen in the interpreter. Or would that be in the compiler? Or maybe the geometry.

Use cases:
* "I want the output to be half the size"
* "I want to make sure the output fits on X surface"

**Comment by johannesnoordanus** [comment]
Yes it can. Use svg2gcode for that. It can scale, translate and rotate. It is also able to handle images.
